doc/examples/reconst_forecast.py

```diff
@@ -0,0 +1,145 @@
+"""
+==============================================================
+Crossing invariant fiber response function with FORECAST model
+==============================================================
+
+We show how to obtain a voxel specific response function in the form of
+axially symmetric tensor and the fODF using the FORECAST model from
+[Anderson2005]_ , [Kaden2016]_ and [Zucchelli2017]_.
+
+First import the necessary modules:
+"""
+
+import matplotlib.pyplot as plt
+from dipy.reconst.forecast import ForecastModel
+from dipy.viz import fvtk
```

Comment:
fvtk will be deprecated in the near future. Can you replace it by the new API ?  You can look at the [Tracking Quick Start example](http://nipy.org/dipy/examples_built/tracking_quick_start.html#example-tracking-quick-start)

Reply:
I'm doing some tests. If I try to do something like:

ren = window.Renderer()
odf_actor = actor.odf_slicer(odf[16:36, :, 30:45], sphere = sphere, colormap='jet')
odf_actor.RotateX(-90)
ren.add(odf_actor)
window.record(ren, out_path='fODFs.png', size=(600, 600), magnification=1)
window.show(ren)

I have a problem: I can see only a line of overlapping ODFs and not a 2D grid of ODFs like while using fvtk. All the visualization parameters are the same for both functions (odf_slicer, and fvtk.sphere_func). 

Is it correct what I'm doing? What is the problem here?

![fodfs](https://user-images.githubusercontent.com/5042250/30809880-913683e4-a203-11e7-917a-000dacdc7980.png)